src/inspector/view/qml/MuseScore/Inspector/notation/notes/internal/NoteheadGroupSelector.qml

```diff
@@ -106,6 +106,10 @@ InspectorPropertyView {
             currentIndex: root.propertyItem && !root.propertyItem.isUndefined
                           ? root.propertyItem.value
                           : -1
+
+            ScrollBar.vertical: StyledScrollBar {
+                    policy: ScrollBar.AlwaysOn
```

Comment:
Could you please fix the indention? It is too big... I'll merge your PR after right this

Reply:
Done!